<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QWidget" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>模式设置</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mode_ComboBox">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>12</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>被动模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>主动模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>助力模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>抗阻模式</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>12</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>方向选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="axis_ComboBox">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>12</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>X轴向</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y轴向</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>69</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标点位置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>127</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_LineEdit">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>127</y>
      <width>156</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>127</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_LineEdit">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>127</y>
      <width>157</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>184</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>力量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="strength_LineEdit">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>184</y>
      <width>156</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>184</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed_LineEdit">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>184</y>
      <width>157</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="speedTest_Btn">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>241</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>速度测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boostTest_Btn">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>241</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>助力测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="againstTest_Btn">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>241</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>阻力测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBack_Btn">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>299</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>回测试点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchLock_Btn">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>299</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>锁定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_Btn">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>299</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startTest_Btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
